build_tools/cmake/build_and_test_tsan.sh

```diff
@@ -47,30 +49,33 @@ CMAKE_ARGS=(
   "-DIREE_HAL_DRIVER_HIP=${IREE_HAL_DRIVER_HIP}"
   "-DIREE_TARGET_BACKEND_CUDA=${IREE_TARGET_BACKEND_CUDA}"
   "-DIREE_TARGET_BACKEND_ROCM=${IREE_TARGET_BACKEND_ROCM}"
+  "-DIREE_HIP_TEST_TARGET_CHIP=${IREE_HIP_TEST_TARGET_CHIP}"
 
   # Workaround for this weird issue:
   # https://github.com/google/benchmark/issues/773#issuecomment-616067912
   "-DRUN_HAVE_STD_REGEX=0"
   "-DRUN_HAVE_POSIX_REGEX=0"
   "-DCOMPILE_HAVE_GNU_POSIX_REGEX=0"
-  "-DCMAKE_CXX_COMPILER_LAUNCHER=sccache"
-  "-DCMAKE_C_COMPILER_LAUNCHER=sccache"
+
+  "-DCMAKE_EXPORT_COMPILE_COMMANDS=ON"
 )
 
+echo "::group::Configuring CMake"
```

Comment:
What does ::group:: do?  Is it an escape sequence interpreted by some terminal emulator?

Reply:
I don't know if it has any function, it's just to match the style of the other scripts